src/Symfony/Component/Form/Extension/Core/DataTransformer/ValueToDuplicatesTransformer.php

```diff
@@ -16,6 +16,8 @@
 
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @implements DataTransformerInterface<mixed, mixed>
```

Comment:
the transformed value is an `array<string, mixed>` (or even better, make this a generic, implementing `DataTransformerInterface<T, array<string, T>>`)